lib/typescript/render/components/UnknownSourceText/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Linkify from 'linkifyjs/react';
+import styles from './index.module.scss';
+
+type TextRenderProps = {
+  text: string;
+  fromContact?: boolean;
+};
+
+export const UnknownSourceText = ({text, fromContact}: TextRenderProps) => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.unknownSourceHeader}>
+        <span className={styles.unknownSource}>Unknown Source</span>
```

Comment:
Could we here perhaps use the `source` identifier instead? After all, many sources will be "known" just without proper rendering logic :)

Reply:
Or just hardcode: "Raw source content"